docs/docs/index.md

```diff
@@ -7,17 +7,71 @@
 
 [![Build](https://github.com/Enedis-OSS/chutney/actions/workflows/build-all.yml/badge.svg?branch=main)](https://github.com/Enedis-OSS/chutney/actions/workflows/build-all.yml){:target="_blank"}
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.chutneytesting/server/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.chutneytesting/server){:target="_blank"}
-[![Zulip chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://chutney-testing.zulipchat.com/){:target="_blank"}
+[![GitHub release](https://img.shields.io/github/v/release/Enedis-OSS/chutney)](https://github.com/Enedis-OSS/chutney/releases/latest){:target="_blank"}
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
 ----
 **Spice up your spec , Better `taste` your app !**
 
 Chutney helps you test deployed applications and validate functional requirements.  
 Chutney provides you a set of capabilities for executing your scenario without having to write and maintain glue code.  
 
-You can find out a comprehensive list of what Chutney can do in the [documentation](documentation/actions).  
-You can learn more about the [concepts](concepts.md) behind.
+:material-rocket-launch: Install, configure, write and run your first scenario [here](/getting_started/requirements.md) !  
+:octicons-book-16: You can find out a comprehensive list of what Chutney can do in the [documentation](/documentation/actions/introduction.md).
+
+----
```

Comment:
Why Choose Chutney ?

Chutney stands out for its simple and flexible approach to test automation, focusing on complex scenarios and end-to-end testing. Compared to other similar tools like Cucumber, Postman, or Karate, here are some key points:

Simplified DSL:
Chutney offers a Kotlin-based Domain-Specific Language (DSL) that allows writing readable and easy-to-maintain scenarios for developpers. This is an attractive alternative for teams preferring a more structured language than the one used by Cucumber with regexp.

Focused on blackbox Testing:
Unlike Cucumber, which is often used for functional testing at build, Chutney is specifically designed for testing on deployed application. It excels at handling API calls, database interactions, and verifying asynchronous call chains.

User-Friendly Interface for Non-Developers:
Chutney includes an interface specifically designed for non-developers, enabling them to execute tests, plan test campaigns, and analyze reports with ease. This sets it apart from other tools like Cucumber or Karate, which often require technical expertise for these tasks. This accessibility bridges the gap between technical and non-technical teams, fostering collaboration and improving efficiency.

Multi-level Gherkin-like language:
Like the glacio (https://github.com/fridujo/glacio) project said :
"
There is a common problem encountered using BDD tools like Cucumber.
This is:
- Non-technical actors write specs
- Developers write glue code containing hidden business specificities
That is an historic clivage which tends to leave grey areas undiscovered until implementation time.
Furthermore, non-technical actors can point out relevant aspects of implementation just as developers can do the same about the business part (aka specs).
As Seb Rose stated in this article (https://claysnow.co.uk/the-testing-iceberg/), levels of readability can differ from the ones encountered in usual test segregation (unit, integration, acceptance, etc.).
"
Chutney’s support for nested steps allows teams to write scenarios that reflect different levels of abstraction.High-level steps can describe functional requirements in a business-readable manner, while the underlying steps handle the technical details. This structured approach emphasizes functional clarity at the top layer, while efficiently managing technical complexity beneath, setting Chutney apart from tools like Cucumber or Karate, which often struggle to balance this granularity.